tests/admin_inlines/tests.py

```diff
@@ -1348,3 +1348,60 @@ def test_inline_formset_error(self):
         self.assertEqual(
             len(self.selenium.find_elements_by_css_selector(tabular_inline_formset_selector)), 1
         )
+
+    def test_inlines_verbose_name(self):
+        """
+        The item added by the "Add another XXX" link shows the correct verbose_name in the inline form.
+        """
+        self.admin_login(username='super', password='secret')
+        params = [
+            'admin:admin_inlines_person_add',
+            'admin:admin_inlines_personproxy_add',
+            'admin:admin_inlines_personproxy1_add',
+            'admin:admin_inlines_personproxy2_add',
+        ]
+        for param in params:
+            with self.subTest(param=param):
+                self.selenium.get(self.live_server_url + reverse(param))
+
+                # There first inline shows the verbose_name
+                self.assertEqual(len(self.selenium.find_elements_by_css_selector(
+                    '.dynamic-fashionista_set')), 1)
+                self.assertEqual(self.selenium.find_elements_by_css_selector(
+                    '.dynamic-fashionista_set')[0].get_attribute('id'), 'fashionista_set-0')
+                self.assertEqual(len(self.selenium.find_elements_by_css_selector(
+                    '.dynamic-fashionista_set#fashionista_set-0 h2')), 2)
+                self.assertEqual(self.selenium.find_elements_by_css_selector(
+                    '.dynamic-fashionista_set#fashionista_set-0 h2')[0].text, 'AVAILABLE LIMITATIONS')
+                self.assertEqual(self.selenium.find_elements_by_css_selector(
+                    '.dynamic-fashionista_set#fashionista_set-0 h2')[1].text, 'CHOSEN LIMITATIONS')
```

Comment:
Do we need all of the test the whole structure? Could we 'just' test these two items? 🤔